pkg/indicator/v2/premium_signal.go

```diff
@@ -0,0 +1,90 @@
+package indicatorv2
+
+import (
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+type PremiumType int
+
+const (
+	Neutral  PremiumType = 0  // no premium
+	Premium  PremiumType = 1  // Base/Quote1 > Base/Quote2
+	Discount PremiumType = -1 // Base/Quote1 < Base/Quote2
+)
+
+type PremiumSignalStream struct {
+	*types.Float64Series
+
+	kLineStream      *KLineStream
+	symbol1, symbol2 string
+	interval         types.Interval
+	price1, price2   float64
+	premiumMargin    float64
+}
+
+func (s *PremiumSignalStream) BackFill(klines []types.KLine) {
+	if s.kLineStream == nil {
+		update1 := types.KLineWith(s.symbol1, s.interval, func(kline types.KLine) {
+			s.price1 = kline.Close.Float64()
```

Comment:
You can use PriceStream if you only need one price from the KLine

Reply:
done.